controller/command/command.go

```diff
@@ -43,13 +43,18 @@ type Validatable interface {
 // system is the leader, apply it locally
 type Dispatcher interface {
 	Dispatch(command Command) error
+	IsLeaderOrLeaderless() bool
```

Comment:
This naming could imply that being a leader is different than being leaderless. 

Just curious what the paths are to becoming leaderless. Is it just that an existing leader can go offline?

Reply:
Yes, I just wanted to account for a state where a leader had gone offline or we were in the middle of an election for anyreason. I figured occasional extra events were better than occasional missed events.